src/traccuracy/_tracking_graph.py

```diff
@@ -201,11 +202,12 @@ def edges(self, limit_to=None):
         Returns:
             dict[tuple[hashable], dict]: A dictionary from edge ids to edge attributes
         """
-        edges = self.graph.edges.items()
         if limit_to is None:
-            return dict(edges)
+            return self.graph.edges
         else:
-            limited_edges = {_id: data for _id, data in edges if _id in limit_to}
+            limited_edges = {
```

Comment:
I think this could be `self.graph.edge_subgraph(limit_to).edges` and then the typing would be the same for both return statements.

Reply:
Nice catch, put this in here 7e72be3. `networkx.DiGraph.edge_subgraph` and `networkx.DiGraph.subgraph` unfortunately don't throw an error when you ask for non-existing things. I do think that this is a reasonable check and added it. This way we also keep the API that you had before.

Reply:
I'm not sure about a good way to make docstrings that include classes from third-party packages, like networkx `OutEdgeView` here. Writing `networkx.classes.reportviews import OutEdgeView` in the docstring seems excessive.

Might be a good segway into typing traccuracy ;), or at least the core parts of it.

Reply:
Yeah would really love to have this package better typed especially once we've sorta settled on the API